src/core_modules/capture-core/metaDataStoreLoaders/optionSets/queries/queryOptionGroups.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { query } from '../../IOUtils';
+
+type Attributes = {
+    page: number,
+    pageSize: number,
+    ids: Array<string>,
+};
+
+const querySpecification = {
+    resource: 'optionGroups',
+    params: (attributes: Attributes) => ({
+        fields: 'id,displayName,options[id],optionSet[id]',
+        filter: `optionSet.id:in:[${attributes.ids.join(',')}]`,
+        page: attributes.page,
+        pageSize: attributes.pageSize,
+    }),
+};
+export const queryOptionGroups = async (ids: Array<string>, page: number, pageSize: number) => {
+    const response = await query(querySpecification, { ids, page, pageSize });
+    return {
+        optionGroups: (response && response.optionGroups) || [],
+        hasNextPage: !!(response && response.pager && response.pager.nextPage),
```

Comment:
Well, I did change one these earlier. But I like the !! better and it's actually faster.

Reply:
Again this is an opinion that you might listen or might not but I believe `!!` is more prone to accidental bugs. For instance if one of us mistakenly removes one of the `!` and we merge it on master then we have a bug. With `Boolean ` this might be avoided.

_I dont mind one or the other, what I mind is that we keep one across our codebase so that we can get used to certain way of writing code and therefore get less tired by reading the codebase._

If you think this is not a compelling argument let me know and I will resolve the conversation.